pages/common/pm2.md

```diff
@@ -23,3 +23,15 @@
 - Restart a process:
 
 `pm2 restart {{myapp}}`
+
+- Dump all processes for resurrecting them later:
```

Comment:
This doesn't make grammatical sense. Perhaps we could do something like this instead?
```suggestion
- Dump all processes for later resurrecting:
```
Also, off choice of words. Usually commands like this use freeze / thaw or something like that instead. "Resurrecting" makes it sound like the process has died previously, which doesn't appear to technically be the case here.

Reply:
Hi @sbrl , I don't mind accepting your suggestion, but I actually took the description from their own help file, **as is** 🤔. What do you suggest?

Reply:
We usually prefer to change the descriptions to make it more suitable to newcomers. But I am curious to know @sbrl why do you think this is grammatically incorrect. It looks fine to me, or perhaps "... to resurrect them later" is also okay.

Reply:
Taking another look, it does actually make sense here. It still sounds weird though - but I guess this is fine as-is.